stl/inc/yvals_core.h

```diff
@@ -906,7 +906,7 @@
 #endif // !defined(_STL_RESTORE_DEPRECATED_WARNING)
 
 #define _CPPLIB_VER       650
-#define _MSVC_STL_VERSION 143
+#define _MSVC_STL_VERSION 145
 #define _MSVC_STL_UPDATE  202504L

```

Comment:
Your PR cites last month's issue #5384, which has already been resolved. You should edit your PR description to cite this month's issue #5461, and as @AlexGuteniev observed, `_MSVC_STL_UPDATE` needs to be updated to May 2025.